build/js/live-editor.output_pjs.js

```diff
@@ -2369,7 +2495,7 @@ window.PJSOutput = Backbone.View.extend({
             this.clear();
 
             // Clear Processing logs
-            this.canvas._clearLogs();
```

Comment:
p5.js doesn't have a _clearLogs method... will need to add this back in when implementing the virtual console